packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/ChatbotHeaderDrawer.tsx

```diff
@@ -31,12 +31,29 @@ const initialConversations: { [key: string]: Conversation[] } = {
   ]
 };
 
+const ERROR = {
+  bodyText: (
+    <>
+      To try again, check your connection and refresh this page. If the issue persists,{' '}
+      <a href="">contact the support team</a>.
```

Comment:
instead of the md syntax, we could also just do a link that doesn't actually go anywhere? So that the text styling is the blue underline/maybe more of a visual cue for consumers?

Reply:
Should have this! I think I had pushed the md format accidentally, but checked and pushed the link after.